### ISSUE TITLE
Anchor link icon is not visible in high contrast mode

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#142]
The anchor link next to each heading is not visible when using Windows High Contrast Mode.

If we resolve #131 by removing these links this issue would also be fixed.

**Comment by andrewhick** [comment]
Discussed with the Tech Docs team and suggest we raise this in the [tech-docs-gem repo](https://github.com/alphagov/tech-docs-gem), but suggest waiting until we've looked at the higher priority WCAG AA issues first (402, 403, 404).
